core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -5632,6 +5632,14 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.CLIENT)
           .build();
+  public static final PropertyKey USER_FILE_METADATA_LOAD_DESCENDANTS_CHECK_DISABLED =
```

Comment:
@tcrain I add a property for the option. But it seems that there is no connection with the current code, or we could make another PR for it.